src/main/java/org/openrewrite/kotlin/internal/KotlinParserVisitor.java

```diff
@@ -1815,64 +1820,58 @@ public J visitProperty(FirProperty property, ExecutionContext ctx) {
                 if (expressions == null) {
                     expressions = new ArrayList<>(2);
                 }
+                List<KtPropertyAccessor> accessors = propertyNodeChildren.stream()
+                    .filter(c -> c instanceof KtPropertyAccessor)
+                    .map(KtPropertyAccessor.class::cast)
+                    .collect(toList());
+                if (accessors.size() > 2) {
+                    throw new RuntimeException("Detected oversized explicit property accessors from a property, should have setter/getter only");
+                }
 
-                int saveCursor = cursor;
-                whitespace();
+                for (KtPropertyAccessor ktPropertyAccessor : accessors) {
+                    Optional<KtDeclarationModifierList> maybeModifier = Arrays.stream(ktPropertyAccessor.getChildren())
+                        .filter(c -> c instanceof KtDeclarationModifierList)
```

Comment:
I'm unsure if these are based on your default IntelliJ settings -- so that you know, our convention is to continue with 8 spaces